gcs-action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'GCS Action'
+description: 'Enable easy upload of files and directories to GCS'
+author: 'GoogleCloudPlatform'
+inputs:
+  service_account_key:
+    description: 'GCP service account key'
+    required: true
```

Comment:
This should not be required. The action should use Application Default Credentials if supplied.

Reply:
Done